docs/src/selectors.md

```diff
@@ -1355,6 +1355,48 @@ await page.Locator("#tsf > div:nth-child(2) > div.A8SBwf > div.RNNXgb > div > di
 await page.Locator("//*[@id='tsf']/div[2]/div[1]/div[1]/div/div[2]/input").ClickAsync();
 ```
 
+### Example
+
+The best selector to use is dependent on the context and the app under test. Here's some questions to keep in mind when authoring selectors:
```

Comment:
Looks like you are pulling the best practices doc into the example section. As in we had nice prioritized talking points and all of a sudden Example decided to take over with its own wisdom. Let's keep points structure as is, we can add more points. And we can provide example with dos and donts for each if you feel like example is helpful!